### ISSUE TITLE
rollup watch mode doesn't work

### ISSUE THREAD

**Issue by jeetiss** [vanilla-extract-css/vanilla-extract#1076]
### Describe the bug

- setup `@vanilla-extract/rollup-plugin` 
- run watch mode `rollup -c -w`
- trigger rebuild
- plugin imports `'styles.css.ts.vanilla.css'` instead of generated files
- `Module not found: Can't resolve 'styles.css.ts.vanilla.css'`

### Reproduction

https://stackblitz.com/edit/node-r8p1k1?file=dist%2Findex.js

### System Info

```shell
System:
  OS: macOS 13.2.1
  CPU: (8) arm64 Apple M1 Pro
  Shell: 5.8.1 
Binaries:
  Node: 18.12.0
  Yarn: 1.22.17
  npm: 8.19.3
Browsers:
  Chrome: 112.0.5615.137
  Firefox: 111.0
  Safari: 16.3
  Safari Technology Preview: 16.4
```

### Used Package Manager

npm

### Validations

- [X] Check that there isn't [already an issue](https://github.com/vanilla-extract-css/vanilla-extract/issues) that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by graup** [comment]
Hi! I tried your repro but it seems to work fine for me. (Run `yarn watch`, then edit the css file. Dist is updated properly.) Am I missing something?

**Comment by graup** [comment]
Did some [more testing](https://stackblitz.com/edit/node-pikheo?file=style.css.js,Button.js,Button.style.css.js,index.js,rollup.config.js,dist%2Findex.js) with two .css.js files and it looks like whenever you save one of them, that one is properly imported, but the import of the other one breaks.

My initial guess is that when rollup rebuild is initiated, we clear all the cached asset import paths (generated according to assetNames), but then only the changed file is regenerated, making other assets point to file names instead of proper import paths.

https://user-images.githubusercontent.com/898549/236717798-f65b3be9-e53a-4098-bb8f-2a1dd1e46fc0.mov

**Comment by jeetiss** [comment]
@graup thanks for the fast fix! Could you please ping me when it would be released

**Comment by graup** [comment]
> @graup thanks for the fast fix! Could you please ping me when it would be released

This was released in @vanilla-extract/rollup-plugin@1.2.2 !
